src/logic/safe/api/fetchSafeTxGasEstimation.ts

```diff
@@ -1,4 +1,8 @@
-import { postSafeGasEstimation, SafeTransactionEstimationRequest } from '@gnosis.pm/safe-react-gateway-sdk'
+import {
+  postSafeGasEstimationV2,
+  SafeTransactionEstimationRequest,
+  SafeTransactionEstimationV2,
```

Comment:
This V2 looks off. Let's rename the old one to `SafeTransactionEstimationV1` and call the new one simply `SafeTransactionEstimation`.

Reply:
Is `SafeTransactionEstimation` v0 or v2? I'd stay as explicit as possible

Reply:
We are only using what is now v2 in frontend. Do you think it would be necessary to keep the 'legacy' function/type? Our docs would only show the v2 response, from my understanding.